### ISSUE TITLE
AttributeError: 'cryptocom' object has no attribute 'safeOrder2'

### ISSUE THREAD

**Issue by oliviera9** [ccxt/ccxt#11105]
- OS: macOS Monterey 12.0.1
- Programming Language version: Python 3.9
- CCXT version: 1.66.5

```
import ccxt.async_support as ccxt  # noqa: E402
import asyncio
import time
import os


async def task():
    crypto = ccxt.cryptocom({
        'apiKey': os.environ.get('KEY'),
        'secret': os.environ.get('SECRET'),
        'verbose': True,  # switch it to False if you don't want the HTTP log
    })
    trading_pair = 'LTC/USDT'
    await crypto.create_limit_buy_order(trading_pair, 0.1, 50)
    await crypto.close()


asyncio.run(task())

```

```
Traceback (most recent call last):
  File "/Volumes/Data/personal/newbot/cryptocom.py", line 18, in <module>
    asyncio.run(task())
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Volumes/Data/personal/newbot/cryptocom.py", line 14, in task
    await crypto.create_limit_buy_order(trading_pair, 0.1, 50)
  File "/Volumes/Data/personal/newbot/venv/lib/python3.8/site-packages/ccxt/async_support/cryptocom.py", line 776, in create_order
    return self.parse_order(result, market)
  File "/Volumes/Data/personal/newbot/venv/lib/python3.8/site-packages/ccxt/async_support/cryptocom.py", line 1250, in parse_order
    return self.safeOrder2({
AttributeError: 'cryptocom' object has no attribute 'safeOrder2'
```

**Comment by kroitor** [comment]
Fixed in 1.66.6+, thanks for pointing it out!

**Comment by kroitor** [comment]
P.S. The new build will arrive in ~20 minutes.
